src/website/webview/urls.py

```diff
@@ -16,4 +23,14 @@
         name="issue_detail",
     ),
     path("github-webhook/", handle_github_hook, name="github_webhook"),
+    re_path(
+        "^affected/(?P<channel>(nixos|nixpkgs)-.*)$",
+        NixderivationPerChannelView.as_view(),
+        name="affected_list_per_channel",
+    ),
+    path(
+        "affected/",
+        RedirectView.as_view(url="nixos-unstable", permanent=True),
+        name="affected_list",
+    ),
```

Comment:
Maybe we should reserve this for an overview page, e.g. listing all stored channels? But this would be for later, so no real change required now.

```suggestion
    # XXX: We may want to put an overview page here
    path(
        "affected/",
        RedirectView.as_view(url="nixos-unstable", permanent=True),
        name="affected_list",
    ),
```

Reply:
Pushed comment. Squashing and merging